libs/public/feature/dashboard/src/lib/components/dashboard/dashboard.component.scss

```
@@ -125,6 +118,7 @@
     }
 
     .application-transaction-id {
+      font-size: rem.fromPx(18px);
```

This font-sizes should be variables. 

```suggestion
      font-size: var(--theme-typography-font-size-large);
```